gitbootcamp.rst

```diff
@@ -331,7 +333,9 @@ Pull requests can be accepted and merged by a Python Core Developer.
 1. At the bottom of the pull request page, click the ``Squash and merge``
    button.
 
-2. Replace the reference to GitHub PR #XXX into GH-XXX.
+2. Replace the reference to GitHub PR #XXX into GH-XXX. If the title
+   too long, add the Pull request to the end of the body as:
+   Pull request: GH-NNN
```

Comment:
It's not necessary to say `Pull request: GH-NNN` in the body. `(GH-NNN)` is what we've been using.
Perhaps it's enough to say:
"If the title is too long, the pull request number can be added to the body."